nss-day-cohort-39/holiday-road-cosmic-chameleons
# rendered weather api to itinerary preview

### Changes

- updated settings files with team members' api keys for fetch access from apis
- changed Weather HTML to reflect data trail returned by weather api 
- added temperature, feels like, highs and lows into forecast preview
- dispatched weather state change event from WDP.js to ItineraryPreview.js
- deleted saved itineraries generated from testing

### Testing

- verify that your .Settings has been copied to a Settings.js file and you have your individual api codes in your copied Settings file
- run json-server -w itineraries.json from api folder
- run json-server -p 8090 -w parks.json from api folder
- select a park from the dropdown 
- verify that the weather forecast appears beneath the park preview
- double check that the other two dropdowns work
- double check that all three details buttons work
- double check that the save itinerary button still works
